docs/development/workflow/maintainer_workflow.rst

```diff
@@ -169,4 +169,61 @@ when a feature was added.  Likewise it lists any features or APIs that were
 changed (and how they were changed) or removed.  It also lists all bug fixes.
 Affiliated packages are encouraged to maintain a similar changelog.
 
+.. _stale-policies:
+
+==============
+Stale Policies
+==============
+
+The ``astropy`` GitHub repository has the following stale policies, which are
+enforced by `action-astropy-stalebot <https://github.com/pllim/action-astropy-stalebot/>`_
+in `.github/workflows/stalebot.yml <https://github.com/astropy/astropy/blob/main/.github/workflows/stalebot.yml>`_
+that runs on a schedule. Hereafter, we refer to this automated enforcer as stale-bot.
+
+All the timing mentioned depends on a successful stale-bot run. GitHub API limits,
+spam protection, or server maintenance could affect the run. The former might
+especially be relevant when there is a significant backlog of stale issues and
+pull requests accumulated.
+
+If you notice unintended stale-bot behaviors, please report them to the Astropy
+maintainers.
+
+Issues
+------
+
+A maintainer applies the "Closed?" label to mark an issue as stale, otherwise it
+stays open until someone manually closes it. Once marked as stale, a warning will
+be issued.
+
+A maintainer can apply "keep-open" label or remove "Closed?" label to remove the
+stale status. Otherwise, stale-bot will close the issue after about a week and
+apply a "closed-by-bot" label.
+
+When both "keep-open" and "Close?" labels exist, the former will take precedence
+and the latter will be removed from the issue.
+
+Pull Requests
+-------------
+
+A pull request becomes stale after about 4-5 months since the last commit (stale-bot
+counts in seconds and naively assumes 30 days per month). When this happens, stale-bot
+applies the "Close?" label to it. A maintainer can also fast-track its staleness by
+manually applying the "Close?" label. Once marked as stale, a warning will be issued.
+
+A maintainer can apply "keep-open" label or remove "Closed?" label to remove the
```

Comment:
Add a note about using `keep-open` very very sparingly, for cases that MUST be kept open. E.g. a PR that cannot be merged in the upcoming release, or other scheduling reasons. Not having time to finish up a PR is generally NOT a good reason to keep it open and let it code rot. After all closed PRs can be reopened and finished up at any time.

Reply:
Ah, that's right. Thanks for reminding me! It is added.